cmd/ocsp-responder/main.go

```diff
@@ -206,12 +200,21 @@ as generated by Boulder's ceremony command.
 		rocspSource, err := redis_responder.NewRedisSource(rocspRWClient, liveSource, liveSigningPeriod, clk, scope, logger)
 		cmd.FailOnError(err, "Could not create redis source")
 
+		var dbMap *db.WrappedMap
+		if c.OCSPResponder.DB != (cmd.DBConfig{}) {
+			// Set DB.DBConnect as a fallback in case DB.DBConnectFile isn't present.
+			c.OCSPResponder.DB.DBConnect = c.OCSPResponder.Source
```

Comment:
We never use the plain `DBConnect` string in prod or dev configs, because we prefer to use DBConnectFile and have the secrets outside of our config. Really the `DBConfig.DBConnect` field should have been deprecated a long time ago.

By the same lights, the `OCSPResponder.Source` field has two modes: one is a `file://` URL (ok), and one is a DBConnect string (not ok, and we don't use this anymore). Similarly the second mode for this field should have been deprecated a long time ago.

Reply:
Would you like those deprecation rolled into this PR? Right now, this line is just a copy-paste from its old location on Line 175.

Reply:
I filed https://github.com/letsencrypt/boulder/pull/6615 to handle DBConnect separately.

Reply:
Aha, I missed that this was moving. I thought it was a newly added fallback. All good to do it in a different PR.